### DISCUSSION TITLE
What's the easiest way to change fill

### DISCUSSION THREAD

**Discussion by kornelijepetak** [ElinamLLC/SharpVectors#295] [Q&A]
I am looking for a way to change the fill color of a svg via XAML.
Either using `SvgViewbox` or `<Image>` with `SvgImage` converter.

The SVG file contains the `<path>` element with no `fill` defined.

Is there an easy non-code-behind way to achieve this?

**Comment by paulushub** [comment]
@kornelijepetak Please can you post a sample SVG. Sometimes it is easier that way, sorry.
Also, are you the one creating them? or what control do you have on the source of the SVG?

**Reply by kornelijepetak** [reply]
Something like this:

```xml
<?xml version="1.0" encoding="utf-8"?>
<svg width="200px" height="200px" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
    <path fill-rule="evenodd" clip-rule="evenodd"
        d="M8 16L3.54223 12.3383C1.93278 11.0162 1 9.04287 1 6.96005C1 3.11612 4.15607 0 8 0C11.8439 0 15 3.11612 15 6.96005C15 9.04287 14.0672 11.0162 12.4578 12.3383L8 16ZM3 6H5C6.10457 6 7 6.89543 7 8V9L3 7.5V6ZM11 6C9.89543 6 9 6.89543 9 8V9L13 7.5V6H11Z" />
</svg>
```

So, the SVG itself does not specify any colors for the stroke or fill (although I'm only interested in the fill).

The whole point is that I want the icon change colors, depending on the external source, driven by the binding, such as `{Binding Path=Fill}` so that when the binding source changes, the icon's color changes too.

Is this possible?
